.github/CODEOWNERS

```diff
@@ -17,6 +18,5 @@
 # AnandatDell (anandrajak1)
 
 
-
 # for all files:
-* @forrestxia @YianZong @baoy1 @taohe1012 @P-Cao @donatwork @harshaatdell @anandrajak1
+* @chimanjain @forrestxia @YianZong @baoy1 @taohe1012 @P-Cao @donatwork @harshaatdell @anandrajak1
```

Comment:
Some of the members could be removes. Forrest, Yian, and Yiming are not longer working on Observability. You should also add Trevor.

Reply:
Done, updated the codeowners.